docs/developer-guide/meta/dist-lock.md

```diff
@@ -0,0 +1,57 @@
+# Distributed Lock
+
+## Introduction
+
+`Meta` provides the functionality of distributed locks via gRPC.
+
+## How to use
+
+The [meta-client][1] crate provides a Rust-based meta client implementation.
+
+[1]: https://github.com/GreptimeTeam/greptimedb/tree/develop/src/meta-client
+
+For example:
+
+```rust
+async fn do_some_work(meta_client: MetaClient) {
+    let name = b"lock_name".to_vec();
+    let expire_secs = 60;
+
+    let lock_req = LockRequest { name, expire_secs };
+
+    let lock_result = meta_client.lock(lock_req).await.unwrap();
+    let key = lock_result.key;
+
+    info!("do some work, take 3 seconds");
+    tokio::time::sleep(Duration::from_secs(3)).await;
+
+    let unlock_req = UnlockRequest { key };
+
+    meta_client.unlock(unlock_req).await.unwrap();
+    info!("unlock success!");
+}
+```
+
+More examples [here][2].
+
+[2]: https://github.com/GreptimeTeam/greptimedb/blob/develop/src/meta-client/examples/lock.rs
+
+Because gRPC is cross-language, it is easy to implement clients for other programming languages.
```

Comment:
@Fengys123 I can not understand the meaning of `cross-language` here. Can you give me some examples of those who also use this word?